lib/resty/core/phase.lua

```diff
@@ -0,0 +1,55 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
```

Comment:
There are a handful of modules that do not adhere to this standard; trying to figure out _which_ style is appropriate to copy is... nebulous and tricky. :) I don't at all mind keeping consistent style (rather glad to see it noted and enforced), but having some dev docs about this kind of thing would be great :D

For reference: 

https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/uri.lua#L4 - all requires and redefs without newline separation
https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/hash.lua#L4 - all requires and redefs without newline separation
https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/exit.lua#L4- all requires and redefs without newline separation

Again: I'm very glad to have a consistent style. And I'd be very happy to put up a PR to fix existing modules that do not adhere to said style. But having it documented so devs have a point of reference aside from guessing from existing context would be awesome.